src/multivariate/hypersphericaluniform.jl

```diff
@@ -0,0 +1,29 @@
+"""
+Uniform distribution on the hyperspherical unit ball in ``d`` dimensions.
+"""
+struct HyperSphericalUniform <: ContinuousMultivariateDistribution
+    d :: Int
+end
+
+# basic properties
```

Comment:
I guess we also have to define `support` and `insupport`.

Reply:
I can do that, but it is not mentioned in the docs as a requirement for multivariate distributions... :)